sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/RetryManager.java

```diff
@@ -97,6 +108,14 @@ Object getResult() {
             .withMaxRetries(maxRetries)
             .withThrottledTimeCounter(throttledTimeCounter)
```

Comment:
Do you think we still need "throttledTimeCounter" for non-quota backoff? Conceptually no but probably good to leave it as is, assuming if there are other server side / network issue related to resource exhausted

Reply:
Agreed